source/dbo/Tables/Report Unmatched Specimens/UnmatchedSpecimensLinkedNotifications.sql

```diff
@@ -1,17 +1,16 @@
 ﻿CREATE TABLE [dbo].[UnmatchedSpecimensLinkedNotifications]
 (
-	[ReferenceLaboratoryNumber] NVARCHAR(50) NOT NULL PRIMARY KEY,
+	[ReferenceLaboratoryNumber] NVARCHAR(50) NOT NULL,
 	[NotificationLinkReason] NVARCHAR(255),
 	[RejectionDate] DATETIME2(7),
 	[User] NVARCHAR(256),
 	[NotificationID] INT,
-	[NotificationStatus] NVARCHAR(30),
-	[NhsNumber] NVARCHAR(10),
-	[BirthDate] DATETIME2(7),
-	[FamilyName] NVARCHAR(35),
-	[GivenName] NVARCHAR(35),
-	[Sex] NVARCHAR (200),
-	[Address] NVARCHAR (150),
-	[Postcode] NVARCHAR(10),
-	[RegionCode] NVARCHAR(50)
+	[Denotified] NVARCHAR(30),
```

Comment:
Is there supposed to be a change to the proc that adds to this table as part of this commit? I don't think denotified is a good name if it's still going to contain the status.

Reply:
I didn't mean to include these changes (they were from a change I made to not join to NTBS directly, but then discarded)